src/platform/user/authentication/config/staging.config.js

```diff
@@ -9,8 +9,12 @@ import {
 
 export default {
   default: {
-    allowedSignInProviders: { ...defaultSignInProviders },
-    allowedSignUpProviders: { ...defaultSignUpProviders },
+    allowedSignInProviders: {
+      ...defaultSignInProviders,
+    },
+    allowedSignUpProviders: {
+      ...defaultSignUpProviders,
+    },
```

Comment:
The `vamock` functionality is only for dev & localhost, staging changes not required.

Reply:
Duly noted. If you look at the commits over time you see that changes made to this file were reverted back to its original functionality and what is highlighted above is the same as before, I will update this file to remove extra lines created.

Reply:
Hi Alex, I've made these changes.